packages/server/src/routes/users.js

```diff
@@ -126,9 +126,9 @@ router.get('/:userId/sendDigestEmail', requireUSDRSuperAdminUser, async (req, re
     }
 
     try {
-        await email.buildAndSendUserSavedSearchGrantDigest(
+        await email.buildAndSendGrantDigestEmails(
             user.id,
-            req.query.date ? moment(new Date(req.query.date)).format('YYYY-MM-DD') : undefined,
```

Comment:
As written, this conversion can change the date string that is passed in:
```
// date will be time 0:00 on 2024-04-23 in UTC time
const date = new Date('2024-04-23');
// momentDate represents same timestamp
const momentDate = moment(date)
// dateString will be the date of the momentDate timestamp in the server's local timezone, which could be '2024-04-23' or '2024-04-22' depending on server's timezone
const dateString = momentDate.format('YYYY-MM-DD')
```

I think the user will be surprised if they call the endpoint with '2024-04-23' and get back grants that open on '2024-04-22', so I changed the conversion function to do everything in the same timezone (local).

Reply:
Good catch!